regolith/helpers/a_proposalhelper.py

```diff
@@ -144,33 +134,27 @@ def db_updater(self):
         cpp_info['other_agencies_submitted'] = rc.other_agencies
         cpp_info['institution'] = rc.institution
         cpp_info['person_months_academic'] = int(rc.months_academic)
-        cpp_info['person_months_summer'] =  int(rc.months_summer)
+        cpp_info['person_months_summer'] = int(rc.months_summer)
         cpp_info['project_scope'] = rc.scope
-        pdoc.update({'cpp_info':cpp_info})
+        pdoc.update({'cpp_info': cpp_info})
         pdoc.update({'currency': rc.currency})
-        if rc.due_date != 'tbd':
+        if rc.due_date:
             pdoc.update({'due_date': date_parser.parse(rc.due_date).date()})
         else:
-            pdoc.update({'due_date': rc.due_date})
-        if rc.duration != 'tbd':
+            pdoc.update({'due_date': 'tbd'})
+        if rc.duration:
             pdoc.update({'duration': int(rc.duration)})
-            if rc.begin_date != 'tbd':
+            if rc.begin_date:
                 begin_date = date_parser.parse(rc.begin_date).date()
-                pdoc.update({'end_date': begin_date + relativedelta(months = int(rc.duration))})
+                pdoc.update({'end_date': begin_date + relativedelta(months=int(rc.duration))})
             else:
                 pdoc.update({'end_date': 'tbd'})
         else:
-            pdoc.update({'duration': rc.duration,
+            pdoc.update({'duration': 'tbd',
```

Comment:
Please could you search for anywhere in the code that uses duration?  if not we may remove it from the db?  Do you think it would be less confusing in the adder to have ``begin_date`` + ``duration`` or ``begin_date`` + ``end_date``, or both maybe, with logic in the code that throws an exception if both are specified and inconsistent?

Reply:
done - I have removed `duration`, so now the code takes in `begin_date` and `end_date` and raises a `ValueError` exception if both are specified and inconsistent

Reply:
can't comment above but capitalisation's changed!